odra-casper/livenet-env/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(once_cell)]
+pub mod livenet_contract_env;
+pub mod livenet_host_env;
+use livenet_host_env::LivenetEnv;
+use odra_core::HostEnv;
+
+pub fn livenet_env() -> HostEnv {
```

Comment:
Everywhere we just call it `env` (in odra, odra-test). The crate name provides additional context.
```suggestion
pub fn env() -> HostEnv {
```

Reply:
Fixed